Update create-wip-policy-using-intune-azure.md (#5393)
MicrosoftDocs/windows-itpro-docs: windows/security/information-protection/windows-information-protection/create-wip-policy-using-intune-azure.md

## Create an Executable rule for unsigned apps

> The executable rule would help you to create an AppLocker rule to sign your unsigned Apps. This would allow you to add the file path or the App publisher contained in the file's digital signature needed for the WIP policy to be applied. 

Reviewer: ```suggestion
The executable rule helps to create an AppLocker rule to sign any unsigned apps. It enables adding the file path or the app publisher contained in the file's digital signature needed for the WIP policy to be applied. 
```

Reviewer 2: Looks good to me. Using the present tense instead of the conditional is perceived as more direct and efficient, unless the material requires the use of conditional phrasing.

Author: Looks good to me too. Thanks
